docs/services/navigation/_index.md

```diff
@@ -115,6 +119,54 @@ The following attributes are available for `Navigation` services:
 | `meters_per_sec` | float | Optional | The default linear velocity for the [base](/components/base/) in meters per second. <br> Default: `0.3` |
 | `obstacles` | obj | Optional | Any obstacles you wish to add to the robot's path. See the [Motion Service](/services/motion/) for more information. |
 
+### Configure and calibrate the Frame System Service for GPS Navigation
+
+{{% alert title="Info" color="info" %}}
+
+The [Frame System Service](/services/frame-system/) is an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
+
+It stores the required contextual information for Viam's services like [Motion](/services/motion/) and [Vision](/services/vision/) to use the position and orientation readings returned by components like [movement sensors](/components/movement-sensor/).
+
+{{% /alert %}}
+
+For the most accurate autonomous GPS navigation with the Navigation Service, configure and calibrate the Frame System Service for the components of your robot.
+To start, add the Frame System Service to your rover [base](/components/base/) and [movement sensor](/components/movement-sensor/).
+
+- Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+  Scroll to the card with the name of your base:
+
+    {{< imgproc src="/services/navigation/click-add-frame-ui.png" alt="The button to add a frame selected with the cursor on the Viam App config builder." resize="500x" >}}
```

Comment:
```suggestion
    {{< imgproc src="/services/navigation/click-add-frame-ui.png" alt="The button to add a frame selected with the cursor on the Viam app config builder." resize="500x" >}}
```
[nit] I know it's alt text and doesn't show up anyway, but this makes it easier when we're sweeping the docs for incorrect capitalization